src/cms/static/js/pages/page_mirroring.js

```diff
@@ -22,12 +22,13 @@ function render_mirrored_page_field(event){
             throw new Error(response.statusText);
         }).then(function (html) {
             // Load rendered field into div and show it
-            u('#mirrored_page_div').html(html);
+            u('#mirrored_page_div').empty();
+            u('#mirrored_page_div').append(html);
             u('#mirrored_page_div').removeClass('hidden');
             u('#mirrored_page_first_div').removeClass('hidden');
         }).catch(function (error) {
             // Show error message instead of field
-            u('#mirrored_page_div').html('<div class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 my-4" role="alert"><p>' + error.message + '</p></div>');
+            u('#mirrored_page_div').append('<div class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 my-4" role="alert"><p>' + error.message + '</p></div>');
```

Comment:
This isn't much different either. Maybe this is "safer":
```suggestion
            u('#mirrored_page_div').html('<div class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 my-4" role="alert"><p id="mirrored-page-error"></p></div>');
            u('#mirrored-page-error').text(error.message);
```
But still, the error message of the fetch function can never return anything which can be considered dangerous.